pages/reference/base-images/base-images.md

```diff
@@ -657,30 +719,185 @@ __Note:__ In the tree diagram, from the bottom to the top, the lower level image
+| {{ $names.company.short }}/orangepi-plus2-fedora | [Docker Hub][armv7hf-orangepi-plus2-fedora-node-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-node-github-link] | latest, 26, 25, 24 |
+| {{ $names.company.short }}/orangepi-plus2-fedora-buildpack-deps | [Docker Hub][armv7hf-orangepi-plus2-fedora-node-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-node-github-link] | latest, 25, 25-scm, 25-curl |
+| {{ $names.company.short }}/orangepi-plus2-fedora-node | [Docker Hub][armv7hf-orangepi-plus2-fedora-node-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-node-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-fedora-node-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-fedora-python | [Docker Hub][armv7hf-orangepi-plus2-fedora-python-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-python-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-fedora-python-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-fedora-golang | [Docker Hub][armv7hf-orangepi-plus2-fedora-golang-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-golang-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-fedora-golang-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-fedora-openjdk | [Docker Hub][armv7hf-orangepi-plus2-fedora-openjdk-dockerhub-link], [github][armv7hf-orangepi-plus2-fedora-openjdk-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-fedora-openjdk-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-github-link] | latest, trusty, xenial, artful, bionic, cosmic |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-buildpack-deps | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-buildpack-deps-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-buildpack-deps-github-link] | latest, jessie, bionic-scm, bionic-curl, xenial, xenial-scm, xenial-curl |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-node | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-node-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-node-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-node-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-python | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-python-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-python-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-python-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-golang | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-golang-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-golang-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-golang-dockerhub-tag-link] |
+
+
+##### BALENA FIN
```

Comment:
Let's make the capitalization here match the rest, Balena Fin

Reply:
fixed